src/platform/plugins/shared/unified_doc_viewer/public/components/doc_viewer_table/table_grid.tsx

```diff
@@ -216,7 +216,7 @@ export function TableGrid({
           {Boolean(warningMessage) && (
             <div>
               <EuiSpacer size="xs" />
-              <EuiCallOut title={warningMessage} color="warning" size="s" />
+              <EuiCallOut announceOnMount title={warningMessage} color="warning" size="s" />
```

Comment:
```suggestion
              <EuiCallOut announceOnMount={false} title={warningMessage} color="warning" size="s" />
```
Not sure on this one, it's visibility is determined when the popover is shown, but not changed after that.